e2e/keywords/common.resource

```diff
@@ -2,37 +2,29 @@
 Documentation       Common keywords
 
 Library             ../libs/keywords/common_keywords.py
+Library             ../libs/keywords/deployment_keywords.py
+Library             ../libs/keywords/network_keywords.py
+Library             ../libs/keywords/recurringjob_keywords.py
```

Comment:
Hi @c3y1huang 
     I want to ask what the purpose is of changing the file naming convention from `snake_case` to `camelCase`, specifically changing the file names from `recurring_job.base` to `recurringjob.base`, and from `persistent_volume_claim_keywords.py` to `persistentvolumeclaim_keywords.py`.

Reply:
The intention is to establish a standardized naming convention for consistency. Of course, we can implement "recurring_job" in alignment with the Kind RecurringJob, and then we will also make a corresponding adjustment to `statefulset` to `stateful_set` for the kind StatefulSet.

It just looks easier to have it in one word instead of having to split it with `_`.